packages/fluentui/docs/src/examples/components/Accordion/Rtl/AccordionExample.rtl.tsx

```diff
@@ -4,12 +4,12 @@ import { Accordion } from '@fluentui/react-northstar';
 const panels = [
   {
     key: 'first',
-    title: 'مرحبا العالم',
+    title: 'مرحبا العالم مرحبًا',
```

Comment:
```suggestion
    title: { key: 'title-1', content: 'مرحبا العالم' },
```

Reply:
Please check one more time the original comment. It's wrong to have a key on an item, keys should exist on `title` and `content`.

https://github.com/microsoft/fluentui/blob/bb553acd81f8a6987ba49370a02e04fe2aaf3755/packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx#L258

```tsx
<Accordion>
  // No panel 🙁
  <Accordion.Title/>
<Accordion>
```

It's extremely confusing, in Semantic UI React is was solved via `AccordionPanel` component that handles keys.

https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/modules/Accordion/AccordionPanel.js

Reply:
I see, confusing indeed... Anyway, fixed!